docs/helm/configure-namespaced-helm-repos.md

```diff
@@ -54,6 +53,49 @@ azure-sample-repo        1m
 
 The addition of the namespace-scoped Helm repository does not impact the behavior of the existing cluster-scoped Helm repository.
 
+## Adding Namespace-scoped Custom Helm Chart Repositories Supporting Basic Authentication
+
+2. To add a new namespace-scoped Helm repositorywhich supports baisc authentication, add a custom Helm Chart Repository CR, for example, `azure-sample-repo` CR to your `my-namespace` namespace:
```

Comment:
You must include [this step](https://github.com/openshift/console/pull/11782#discussion_r952446003) here.